pages/photos.vue

```diff
@@ -1,18 +1,27 @@
 <template>
   <UContainer class="space-y-4">
     <ClientOnly>
-      <UButton
-        :label="
-          photos.length === 0
-            ? $t('photos.loadOrRefreshButton.loadButton')
-            : $t('photos.loadOrRefreshButton.refreshButton')
-        "
-        :disabled="!validS3Setting"
-        variant="outline"
-        @click="listPhotos"
-      />
+      <div class="flex gap-4 justify-between">
```

Comment:
Use `form` instead of `div` can have better a11y (such as enter to submit). Maybe you can use UForm without using its validation.